pkg/pods/pod.go

```diff
@@ -292,17 +239,7 @@ func (pod *Pod) Uninstall() error {
 		}
 	}
 
-	// remove runit services
-	sbTemplate := runit.NewSBTemplate(pod.Id)
-	err = pod.ServiceBuilder.Remove(sbTemplate)
-	if err != nil {
-		return err
-	}
-
-	_, err = pod.ServiceBuilder.Rebuild()
-	if err != nil {
-		return err
-	}
+	// remove runit services here
```

Comment:
good question... I removed the old code after eliminating those methods but it should probably delete the service and prune(). will fix